packages/components/app/styles/components/alert.scss

```diff
@@ -6,10 +6,38 @@
 //
 
 .hds-alert {
+  align-content: flex-start;
   background-color: var(--token-color-surface-warning);
-  padding: 1.25rem; // 20px
   display: flex;
-  align-content: flex-start;
+  padding: 1.25rem; // 20px
+
+  code, pre {
```

Comment:
[issue] this should be specified more, using the specific "parent" container to make sure we're targeting the right elements. something like this:
```
...
    .hds-alert__description {
        code, pre {
            [your styles here]
        }
    }
...
```